scripts/jenkins-yoctobuild-build.sh

```diff
@@ -156,27 +188,22 @@ then
     mender-artifact write rootfs-image -t vexpress-qemu -n test-update -u $BUILDDIR/tmp/deploy/images/vexpress-qemu/core-image-full-cmdline-vexpress-qemu.ext4 -o successful_image_update.mender
     # run tests on qemu
     if [ "$TEST_QEMU" = "true" ]; then
-        if [ ! -z "$PR_TO_TEST" ]; then
-            START_COMMENT="##### QEMU acceptance tests [job]($BUILD_URL) starting for:\nHEAD commit: $GIT_COMMIT"
-            github_pull_request_comment "$START_COMMENT"
-        fi
 
         HTML_REPORT="--html=report.html --self-contained-html"
         if ! pip list|grep -e pytest-html >/dev/null 2>&1; then
             HTML_REPORT=""
             echo "WARNING: install pytest-html for html results report"
         fi
 
-        py.test --verbose --junit-xml=results.xml $HTML_REPORT || FAILED=1
+        py.test --verbose --junit-xml=results.xml $HTML_REPORT || QEMU_TESTING_STATUS=1
```

Comment:
Use `QEMU_TESTING_STATUS=$?` instead. In most cases it doesn't matter, but there are some cases where keeping the return code is useful.